CheckDrive.Web/CheckDrive.Web/Views/Cars/Create.cshtml

```diff
@@ -2,105 +2,84 @@
+            <span asp-validation-for="ManufacturedYear" class="text-danger"></span>
+        </div>
+        <div class="col-md-6 mb-3">
+            <label asp-for="Mileage" class="control-label">Bosib o'tilgan masofasi [km]</label>
+            <input asp-for="Mileage" class="form-control" />
+            <span asp-validation-for="Mileage" class="text-danger"></span>
+        </div>
+        <div class="col-md-6 mb-3">
+            <label asp-for="FuelTankCapacity" class="control-label">Yoqilg'i baki sig'imi</label>
+            <input asp-for="FuelTankCapacity" class="form-control" id="fuelTankCapacity" />
+            <span asp-validation-for="FuelTankCapacity" class="text-danger"></span>
+        </div>
+        <div class="col-md-6 mb-3">
+            <label asp-for="MeduimFuelConsumption" class="control-label">100 km uchun yoqilg'i me'yori [litr]</label>
+            <input asp-for="MeduimFuelConsumption" class="form-control" id="meduimFuelConsumption" />
+            <span asp-validation-for="MeduimFuelConsumption" class="text-danger"></span>
+        </div>
+        <div class="col-md-6 mb-3">
+            <label asp-for="RemainingFuel" class="control-label">Avtomobildagi yoqilg'i qoldig'i [litr]</label>
+            <input asp-for="RemainingFuel" class="form-control" id="remainingFuel" oninput="validateRemainingFuel()" />
+            <span asp-validation-for="RemainingFuel" class="text-danger"></span>
+        </div>
+        <div class="col-md-6 mb-3">
+            <label asp-for="OneYearMediumDistance" class="control-label">1 yillik bosib o'tish me'yori [km]</label>
+            <input asp-for="OneYearMediumDistance" class="form-control" id="oneYearMediumDistance" oninput="calculateMonthlyDistance()" />
+            <span asp-validation-for="OneYearMediumDistance" class="text-danger"></span>
+        </div>
+        <div class="col-md-6 mb-3">
+            <label for="monthlyDistance" class="control-label">1 oylik bosib o'tish me'yori [km]</label>
+            <input type="text" id="monthlyDistance" class="form-control" readonly />
+        </div>
+        <div class="col-md-6 mb-3">
+            <label for="yearlyMediumFuel" class="control-label">1 yillik yoqilg'i me'yori [litr]</label>
+            <input type="text" id="yearlyMediumFuel" class="form-control" readonly />
+        </div>
+        <div class="col-md-6 mb-3">
+            <label for="monthlyMediumFuel" class="control-label">1 oylik yoqilg'i me'yori [litr]</label>
+            <input type="text" id="monthlyMediumFuel" class="form-control" readonly />
+        </div>
     </div>
-</div>
+    <div class="modal-footer d-flex justify-content-center ">
+        <button type="submit" class="btn btn-success" style="width:200px" id="saveCarBtn">
+            <i class="fa fa-plus"></i> Yaratish
+        </button>
+    </div>
+</form>
 
+<script src="~/lib/jquery-validation-unobtrusive/jquery.validate.unobtrusive.min.js"></script>
```

Comment:
All Scripts be in wwwroot or layout.cshtml , in my opinion

Reply:
Of course it is, but I put it because jquery didn't work